app/forms/work_form.rb

```diff
@@ -113,9 +113,7 @@ def sync(*)
                end
                return item if item
 
-               new_attached_file = AttachedFile.new
-               new_attached_file.file.attach(ActiveStorage::Blob.find_signed(fragment['file']))
-               attached_files.append(new_attached_file)
+               attached_files.append(AttachedFile.new)
```

Comment:
This code was needed when I added it. If it's no longer needed, can you point to the change that obviated the need? i.e., how/where do we make the connection between the AF instance and the AS::Blob?

Reply:
@mjgiarlo reform does this automatically when it syncs the `file` attribute from the form to the `file` attribute on AttacheFile.  The `file` attribute on attached file is generated via the has_one_attached method.  See https://edgeguides.rubyonrails.org/active_storage_overview.html#has-one-attached.

Reply:
@jcoyne This doesn't seem to work for me. Pair?